build/TestLocalBuild.CMD

```diff
@@ -1,15 +1,30 @@
 @echo off
 setlocal ENABLEEXTENSIONS
 set VSLANG=1033
-SET DOTNET_CLI_UI_LANGUAGE = en-US
-chcp 437
-
+SET DOTNET_CLI_UI_LANGUAGE=en-US
+IF NOT DEFINED UNO_UWP_BUILD (SET UNO_UWP_BUILD=true)
```

Comment:
This should not be needed, as it's defined here: https://github.com/unoplatform/uno/blob/746d6867c3d21bdb355d48d8fb70ccc2cb9be879/src/Directory.Build.props#L41.

What's the error raised without it?